include/z64camera.h

```diff
@@ -894,7 +894,7 @@ typedef struct {
     /* 0x28 */ f32 eyeLerp;
     /* 0x2C */ s16 eyeLerpPhase;
     /* 0x2E */ s16 isSfxOff;
-    /* 0x30 */ s16 yaw;
+    /* 0x30 */ s16 forwardsYaw; // yaw to the forwards crawling direction
```

Comment:
"forwardYaw" sounds more natural to me. looking it up it seems that the differences between the two are pretty muddy and can be used interchangeably, so its not a big deal wither way

Reply:
I didn't even notice that I also did the reverse order of what was suggested, `yawForward` vs. `forwardYaw` (vs.  `yawForwards` vs. `forwardsYaw`), I'm going to take a hunch that it's not consistent

Reply:
forwardYaw sounds even better to me personally, but yeah whatever works